submissions.md

```diff
@@ -3,18 +3,63 @@ layout: default
 title: "ProbNum25 : Submissions"
 ---
 
-# Proceedings of ProbNum25
+
+# Call for Papers
+
+_Full details, including style files, will be published here in due course._
+
+## Proceedings of ProbNum25
 
 ProbNum25 will have a proceedings issue, published in the [Proceedings of Machine Learning Research (PMLR)](https://proceedings.mlr.press).
 Talks at ProbNum25 will be selected from among the submissions.
 Submissions can be up to 8 pages, with subjects broadly in the area of probabilistic computation.
 We also welcome and encourage shorter papers.
 
+## Important dates
 Timescales for submitting are anticipated to be as follows, though these are subject to revision:
 
 - **Submission Deadline** 5th March 2025.
 - **Reviews Released** 9th April 2025.
 - **Responses Due** 18 April 2025.
 - **Decisions** 14 May 2025.
 
-Full details, including style files, will be published here in due course.
+
+## Subject areas
+
+We welcome submissions in areas including but not limited to:
+
+- Probabilistic numerical methods
+- Applications of probabilistic numerics
+- Decision-making under uncertainty
+- Uncertainty quantification
```

Comment:
- Uncertainty quantification in computation, and computational methods for efficient uncertainty quantification ?